Source/Core/Core/Boot/AncastTypes.h

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+#include "Common/Crypto/SHA1.h"
+
+// Magic number
+constexpr u32 ANCAST_MAGIC = 0xEFA282D9;
+
+// The location in memory where PPC ancast images are booted from
+#define ESPRESSO_ANCAST_LOCATION_PHYS 0x01330000
```

Comment:
```suggestion
constexpr u32 ESPRESSO_ANCAST_LOCATION_PHYS = 0x01330000;
```

Reply:
Oops missed those, only ended up fixing the first one.